spec/tutum_spec.rb

```diff
@@ -2,51 +2,94 @@
 
 test_username = ENV['TUTUM_USERNAME']
 test_api_key = ENV['TUTUM_API_KEY']
+test_tutum_auth = "ApiKey #{test_username}:#{test_api_key}" #best way to handle this?
```

Comment:
The only reason I did this is because if you don't provide a tutum_auth then tests would fail. And there is no official way of getting a tutum_auth without setting up a temporary service to get one from?
maybe:
```ruby
test_tutum_auth = ENV['TUTUM_AUTH'] || "ApiKey #{test_username}:#{test_api_key}"
```